NebulaHost/PacketProcessors/Factory/Inserter/NewSetInserterPickTargetProcessor.cs

```diff
@@ -0,0 +1,26 @@
+﻿using NebulaModel.Attributes;
+using NebulaModel.Networking;
+using NebulaModel.Packets.Factory.Inserter;
+using NebulaModel.Packets.Processors;
+using NebulaModel.DataStructures;
+using NebulaWorld.Factory;
+
+namespace NebulaHost.PacketProcessors.Factory.Inserter
+{
+    [RegisterPacketProcessor]
+    class NewSetInserterPickTargetProcessor : IPacketProcessor<NewSetInserterPickTargetPacket>
+    {
+        public void ProcessPacket(NewSetInserterPickTargetPacket packet, NebulaConnection conn)
+        {
+            PlanetFactory factory = GameMain.data.factories[packet.FactoryIndex];
+            if (factory != null)
+            {
+                FactoryManager.TargetPlanet = factory.planetId;
+                factory.WriteObjectConn(packet.ObjId, 1, false, packet.OtherObjId, -1);
+                factory.factorySystem.SetInserterPickTarget(packet.InserterId, packet.OtherObjId, packet.Offset);
+                factory.entityPool[packet.ObjId].pos = packet.PointPos.ToVector3();
+                FactoryManager.TargetPlanet = -2;
```

Comment:
what is -2?

Reply:
`-2` is the default value. Since I am using comparison: `FactoryManager.TargetPlanet == GameData.mainPlayer.planetId` and I want to prevent the situation when player is in space and `planetId` is `-1`. In that case, this comparison should be also false.

Reply:
Constants should be named then. const int InSpace = 2.

Reply:
Added constant for the `PLANET_NONE = -2` :)

Reply:
```cs
FactoryManager.TargetPlanet = -2;
//Replaced with:
FactoryManager.TargetPlanet = FactoryManager.PLANET_NONE;
```